src/js/pensions/config/form.js

```diff
@@ -216,16 +215,116 @@ const formConfig = {
             })
           }
         },
-        // dependentsNetWorth: {
-        //   path: '/financial-disclosure/net-worth/dependents/household/:index',
-        //   title: 'Net worth',
-        //   arrayPath: 'childrenInHousehold',
-        //   itemFilter: (item) => item.childNotInHousehold,
-        //   component: ArrayPage,
-        //   initialData: {
-        //     childrenInHousehold: [{}]
-        //   }
-        // }
+        dependentsNetWorth: {
+          path: 'financial-disclosure/net-worth/dependents/:index',
+          title: item => `${item.childFullName.first} ${item.childFullName.last} net worth`,
+          showPagePerItem: true,
+          arrayPath: 'children',
+          itemFilter: (item) => !item.childNotInHousehold,
+          initialData: {
```

Comment:
Yeah. I think I'll clean that up in a later PR, since it's easier to have some of this in the form until we get some of the earlier pages built.